ppi.c

```diff
@@ -81,6 +81,37 @@ setup_ppi_variables(void)
         free(data);
     }
 
+    ppi_locked = 0;
+    status = internal_set_variable(TCG2_PHYSICAL_PRESENCEFLAGSLOCK_NAME,
+                                   TCG2_PHYSICAL_PRESENCEFLAGSLOCK_NAME_SIZE,
+                                   &gEfiTcg2PpiXenGuid,
+                                   &unlocked,
+                                   sizeof(uint8_t),
+                                   ATTR_BRNV);
+    if (status != EFI_SUCCESS) {
+        ERR("internal_set_variable(PPILock) returned 0x%016lx\n", status);
+        return false;
+    }
+
+    ppi_locked = false;
```

Comment:
This was already set above in line 84 - but it can be removed anyway as I suggested earlier.